src/common/buffer.hpp

```diff
@@ -284,104 +284,85 @@ class RxCounters {
     }
 };
 
-class Frame_stats {
+/**
```

Comment:
Let's remove the `//` in the comment block and fit it to 80 chars. We could call them "dependency counters" instead of "metrics" in the documentation.